index.js

```diff
@@ -1,5 +1,6 @@
 const axios = require('axios')
 const pluralize = require('pluralize')
+const _ = require('lodash')
```

Comment:
At the Ember meetup tonight someone brought up the fact that we are super concerned about our browser apps being under 250kb, meanwhile the average iOS app is well over 50MB. I think we can handle the 22k.